### PR TITLE
updated README.md : Closes #10

### LINKED ISSUE #10: [DOCS] Readme
### Description

Create Readme for the project

- What's the project for
- features
- how to run it on local
- and many more things

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by nnisarggada**
## Fixes #10 

Closes #10

## Changes proposed

Updated README.md

## Check List

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,87 @@
...
+Frontend Freaks is a community-driven platform dedicated to frontend development. Our goal is to provide resources, tutorials, articles, and tools that empower frontend developers to excel in their craft. Whether you are interested in HTML, CSS, JavaScript, or any other frontend technology, you'll find valuable content here to enhance your skills.
+
+## Tech Stack
+
+The Frontend Freaks website is built using the following technologies:
+
+- [NEXT.js](https://nextjs.org/): A popular React framework for building fast, modern web applications.
+- [Tailwind CSS](https://tailwindcss.com/): A utility-first CSS framework that helps you quickly design and style your web components.
+- [Node.js](https://nodejs.org/): A JavaScript runtime for server-side development.
+- [npm](https://www.npmjs.com/): The package manager for JavaScript that manages project dependencies.
```
Comment (ManishBisht777) [inline README.md:25]: add contentlayer as well

**Review by ManishBisht777 — COMMENTED** [review]
Great work @nnisarggada

Just a little thing mentioned in inline comments

File: README.md (hunk 1 above)
Comment (nnisarggada) [reply README.md:26]: updated :D

Comment (ManishBisht777) [reply README.md:26]: Thanks for the super super fast updates :)

**Review by ManishBisht777 — APPROVED** [review]
LGTM and again good work :)

**Pull request merged**